app/assets/stylesheets/custom.scss

```
@@ -18,10 +18,29 @@ text-align: center;
 
 .center h1 {
 margin-bottom: 10px;
+padding-top: 60px;
+}
+
+section {
+overflow: auto;
```

indent
fix the same error in this file
```suggestion
  overflow: auto;
```

ex:
```
#Good
textarea {
  resize: vertical;
}
```